PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,17 @@
 ## Description
-<!-- Please explain the changes you made here. -->
+
 
 ## Testing done
-<!-- Please explain local testing done, this can include
-automated testing, manual testing, cross browser testing -->
 
-## Testing Plan
-<!-- Please explain testing planned this can include
-automated testing, manual testing, cross browser testing -->
 
 ## Screenshots
-<!-- Please provide screenshots of UI changes if applicable -->
-
-## Acceptance Criteria (Definition of Done)
 
-#### Unique to this PR
-<!-- This would be a good place to include feature flag check item and info, specific dashboards and instrumentation check item and info -->
-- [ ] _Replace this line with individual tasks unique to your PR_
 
-#### Applies to all PRs
+## Acceptance criteria
+- [ ]
 
-- [ ] Appropriate logging
+## Definition of done
```

Comment:
Feature flags should be added to this list. We don't have the most robust feature flag system right now, but all new major functionality should be wrapped in feature flags.

Reply:
(feature specific, and not build type, unless the build type _is_ the feature)

Reply:
As far as I know, we don't have feature flag capabilities on vets-website. I've been really interested in making that a thing, but it isn't as yet.

To that end, I just added that to our front end sync agenda. :wink:

Until we have that capability, it may not make sense to add that to this list yet.